wp-content/plugins/core/src/Query/Search.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tribe\Project\Query;
+
+class Search {
+	/**
```

Comment:
Once you merge in `main` and run `yarn install; lefthook install -a` phpcs should catch the formatting issues in this file.